src/systems/camera-system.js

```diff
@@ -86,33 +132,79 @@ export class CameraSystem {
 
   inspect(o) {
     if (this.mode !== CAMERA_MODE_INSPECT) {
-      this.preInspectMode = this.mode;
+      this.snapshot.mode = this.mode;
     }
     this.mode = CAMERA_MODE_INSPECT;
     this.inspected = o;
+
+    (getBatch(o, this.batchManagerSystem) || o).traverse(enableInspectLayer);
```

Comment:
i'm confused by this, won't this put everything in the batch into the layer, not just the inspected object? i'm not familiar with the batching code

Reply:
i guess in practice it doesn't do that, but can you explain why if it's obvious?

Reply:
oh, i see, you clobber the matrix :)

Reply:
Yes -- Made it hide all the other batched objects by setting their (batched) transforms